virttest/qemu_storage.py

```diff
@@ -1554,11 +1554,31 @@ def measure(self, target_fmt, size=None, output="human"):
         """
         cmd_list = [self.image_cmd, "measure", ("--output=%s" % output),
                     ("-O %s" % target_fmt)]
+
+        if target_fmt == "luks":
+            target_image = self.params.get("image_measure_target", "tgt")
+            target_image_params = self.params.object_params(target_image)
+            target_image_params["image_format"] = "luks"
+            target_image_params["image_secret"] = "measure"
```

Comment:
As we talked,  if the caller didn't define the `image_measure_target` and corresponding parameters in the cfg, will use the default configure parameters: `target_image_params["image_format"] = "luks"` and `target_image_params["image_secret"] = "measure"`, otherwise use the caller costomizied parameters.  But from the above code, if the caller define them in the cfg, like:
```
image_measure_target = yhong
image_format_yhong = luks
image_secret_yhong = yhong_measure
```
then the `image_secret_yhong = yhong_measure` will not work since the` target_image_params["image_secret"] = "measure"`.
So from the above codes, the caller only could define the image tag name, can't define their other attributes, like `secret`.
Maybe I misunderstood the point we talked about, please correct me, thanks.

Reply:
Let's also set the default value for image_secret,  like below codes.  @YongxueHong Do you agree?
```
target_image = self.params.get("image_measure_target", "tgt")
target_image_secret = self.params.get("image_secret_%s" % target_image, "measure")
target_image_params = self.params.object_params(target_image)
target_image_params["image_format"] = "luks"
target_image_params["image_secret"] = target_image_secret
```

Reply:
> Let's also set the default value for image_secret, like below codes. @YongxueHong Do you agree?
> 
> ```
> target_image = self.params.get("image_measure_target", "tgt")
> target_image_secret = self.params.get("image_secret_%s" % target_image, "measure")
> target_image_params = self.params.object_params(target_image)
> target_image_params["image_format"] = "luks"
> target_image_params["image_secret"] = target_image_secret
> ```

Hi @XueqiangWei 
Okay, looks good to me.
Thanks.